pkg/restapi/oidc/operations.go

```diff
@@ -465,10 +474,20 @@ func (o *Operation) onboardUser(sub string) error {
 		return fmt.Errorf("create operational keystore : %w", err)
 	}
 
+	var userSDSVaultURL string
+
+	if o.userSDSClient != nil {
+		userSDSVaultURL, err = createSDSDataVault(o.userSDSClient, controller)
```

Comment:
The vault ID needs to be stored in bootstrap store and probably somewhere locally so that the aries framework's EDV provider can be initialized with it.

Reply:
Here is the [TODO](https://github.com/trustbloc/edge-agent/issues/489).

Reply:
Oh ok. Will you do that next?

Reply:
Yes